licensecheck.properties

```diff
@@ -0,0 +1,25 @@
+category=Languages
+name=License Check
+description=Checks Maven, Gradle and NPM project for license violations
+issueTrackerUrl=https://github.com/porscheinformatik/sonarqube-licensecheck/issues
+devVersion=5.0.0-beta-2
+publicVersions=4.0.2
+archivedVersions=3.2.0
+
+defaults.mavenGroupId=at.porscheinformatik.sonarqube.licensecheck
+defaults.mavenArtifactId=sonarqube-licensecheck-plugin
+
+5.0.0-beta-2.description=JavaScript-only and Groovy-only projects, configuration uses default functionality
+5.0.0-beta-2.sqVersions=[8.9,LATEST]
+
+4.0.2.description=fix: create license with status disallowed
+4.0.2.sqVersions=[8.4,8.9.*]
+4.0.2.downloadUrl=https://github.com/porscheinformatik/sonarqube-licensecheck/releases/download/v4.0.2/sonarqube-licensecheck-plugin-4.0.2.jar
+4.0.2.date=2021-05-04
+4.0.2.changelogUrl=https://github.com/porscheinformatik/sonarqube-licensecheck/releases/tag/v4.0.2
+
+3.2.0.description=Gradle support, NPM transitive check config
```

Comment:
There's no point in publishing a version straight to archived status. This should be removed.